frontend/src/locales/pt/translation.json

```diff
@@ -168,6 +168,7 @@
     "ChartTypeError": "Selecione um tipo de gráfico",
     "DisplayHorizontalScroll": "Exibir com rolagem horizontal",
     "ShowDataLabels": "Mostrar rótulos de dados",
+    "ComputePercentages": "!!NEED TRANSLATION!!",
```

Comment:
We prefer not to change the es and pt translation files, only the English one, that way is easier for us later.